system/ee/ExpressionEngine/Controller/Channels/Channels.php

```diff
@@ -1255,6 +1255,19 @@ private function renderSettingsTab($channel, $errors)
             )
         );
 
+        if (IS_PRO && ee('pro:Access')->hasValidLicense() && (ee()->config->item('enable_entry_cloning') === false || ee()->config->item('enable_entry_cloning') === 'y')) {
```

Comment:
In this section:

`ee()->config->item('enable_entry_cloning') === false`

I believe the idea is that we're checking that the config item is not set. Is there any chance that it would return `false` if it were set to `n` or something like that? I don't think the config helper would do that, but this did seem like an odd check until I realized what it probably was for. 

At the very least, a comment would help with understanding the logic a bit.

Reply:
`=== false` is specifically checking if the value is not set at all

Reply:
There is no way for `n` to convert to `false` here

Reply:
Okay, that's what I figured, but I wanted to make sure.